### PR TITLE
add encrypt-data-using-fakem-cipher

### PR DESCRIPTION
**Pull request by mike-hunhoff**

### CODE REVIEW

File: nursery/encrypt-data-using-fakem-cipher.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: encrypt data using FAKEM cipher
```
Comment (williballenthin) [inline nursery/encrypt-data-using-fakem-cipher.yml:4]: is FAKEM a family or encryption algorithm?

**Review by williballenthin — APPROVED** [review]

File: nursery/encrypt-data-using-fakem-cipher.yml (hunk 1 above)
Comment (mike-hunhoff) [reply nursery/encrypt-data-using-fakem-cipher.yml:4]: `FAKEM` is a malware family - I'll add a description to make this more clear.

**Review by Ana06 — APPROVED** [review]
LGTM 👍 The tests are failing due to the Python 3 update in capa. This should fix it: https://github.com/fireeye/capa-rules/pull/315

File: nursery/encrypt-data-using-fakem-cipher.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  features:
+    - and:
+      - characteristic: tight loop
+      - count(mnemonic(ror)): 5
+      - count(mnemonic(xor)): 5
+      - number: 0x59
+      - number: 0x48
+      - number: 0x43
+      - number: 0x52
+      - number: 0x41
```
Comment (mr-tz) [inline nursery/encrypt-data-using-fakem-cipher.yml:19]:
```suggestion
      - number: 0x59 = Y
      - number: 0x48 = H
      - number: 0x43 = C
      - number: 0x52 = R
      - number: 0x41 = A
```

File: nursery/encrypt-data-using-fakem-cipher.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: encrypt data using FAKEM cipher
+    namespace: data/encryption
+    author: michael.hunhoff@fireeye.com
+    description: Detect custom encryption cipher used by FAKEM malware family
+    scope: basic block
+    references: https://attack.mitre.org/software/S0076/
```
Comment (mr-tz) [inline nursery/encrypt-data-using-fakem-cipher.yml:9]:
```suggestion
    scope: basic block
    att&ck:
      - Defense Evasion::Obfuscated Files or Information [T1027]
    mbc:
      - Cryptography::Decrypt Data [C0031]
      - Cryptography::Encrypt Data [C0027]
    references: https://attack.mitre.org/software/S0076/
```

**Review by mr-tz — APPROVED** [review]
looks good! just adding some suggestions

**Comment by mr-tz** [comment]
branch needs to be rebased to use pass linting per Ana's comment and fix

File: nursery/encrypt-data-using-fakem-cipher.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: encrypt data using FAKEM cipher
+    namespace: data/encryption
```
Comment (re-fox) [inline nursery/encrypt-data-using-fakem-cipher.yml:5]: should this be `data-manipulation/encryption`?

**Comment by mr-tz** [comment]
bump

**Pull request merged**